pkg/k8s/apis/cilium.io/utils.go

```diff
@@ -49,6 +49,14 @@ func ExtractNamespace(np *metav1.ObjectMeta) string {
 	return np.Namespace
 }
 
+// GetPolicyLabels returns a LabelArray with for the given namespace and name.
```

Comment:
returns a LabelArray for the given namespace and name

Reply:
Done